activation/activation.go

```diff
@@ -14,7 +14,7 @@ import (
 
 const AtxProtocol = "AtxGossip"
 
-var activesetCache = NewActivesetCache(1000)
+var activesetCache = NewActivesetCache(100)
```

Comment:
Better leave as it was.
We don't want to play with these numbers right now. We know it works with 1000. 100 might be too low.

Reply:
@barakshani please tell @gavraz what params you are actually using

Reply:
I ran with 100, but we can stick with 1000. The items in this cache are very small (hash,integer).

Reply:
tov